content/en/docs/KRE/user/50_krt_v2/specs/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "KRT V2 YAML specs"
+description: >
+  YAML Definition of a KRT file.
+weight: 20
+---
+
+A KRT yaml file is a declarative file describing the content of the KRT.
+It has general description of the Version, a GRPC entrypoint for the Runtime Version and
+nodes that are connected between each other to form workflows that will be access through GRPC services defined on the entrypoint.
+
+Example below is directly taken from our krt v2 demo repo:
+
+```yaml
+version: classificator-v1
+krtVersion: v2 # new tagged krt version
+description: Demo email classificator for branching features.
+entrypoint:
+  proto: public_input.proto
+  image: konstellation/kre-entrypoint:latest
+config:
+  variables:
```

Comment:
It would be interesting to add some examples of variables and file names here.

Reply:
Agreed, I'll add some